docs/docs/mdx/writing-pages.md

```diff
@@ -67,47 +84,126 @@ The chart is rendered inside our MDX document.
 <FAQ />
 ```
 
-## Exports
+The `<Chart />` component coming from a `.js` file would be written like any
+other React component, while the `<FAQ />` component coming from an `.mdx`
+file might look something like this:
+
+<!-- prettier-ignore -->
+```markdown:title=src/components/faq.mdx
```

Comment:
Can we change all of these `markdown` to `mdx`? May take some looking at the way these get added -- if it's not a recognized extension I think it defaults to `text` or something.

Reply:
I'll look into this yeah that would be good, I think I briefly checked what it would take but never really bothered.

Reply:
Thanks for noting this, I thought this might be tough, turns out is was pretty easy to fix once I went looking for where it was happening:

![image](https://user-images.githubusercontent.com/21114044/59540904-6c17b680-8ebc-11e9-82a1-b056fca86c27.png)